articles/openstack/ostack-how-basic-heat-template.md

```diff
@@ -0,0 +1,164 @@
+      external_gateway_info: { network: internet }
+      name: heat_router
+  heat_router_interface:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: heat_router }
+      subnet: { get_resource: heat_network_subnet }
+  heat_server_port:
+    type: OS::Neutron::Port
+    properties:
+      network: { get_resource: heat_network }
+      fixed_ips:
+        - subnet_id: { get_resource: heat_network_subnet }
+  heat_server:
+    type: OS::Nova::Server
+    properties:
+      name: heat_server
+      flavor: { get_param: flavor }
+      image: { get_param: image }
+      networks:
+        - port: { get_resource: heat_server_port}
+  heat_server_public_ip:
+    type: OS::Neutron::FloatingIP
+    properties:
+      floating_network: "internet"
+  heat_server_ip_assoc:
+    type: OS::Neutron::FloatingIPAssociation
+    properties:
+      floatingip_id: { get_resource: heat_server_public_ip }
+      port_id: { get_resource: heat_server_port }
+
+outputs:
+  heat_server_public_ip:
+    description: IP Address of the deployed heat_server instance
+    value: { get_attr: [ heat_server_public_ip, floating_ip_address ]}   
+```
+
+## Deploying the stack
+
+> [!NOTE]
+> You will need to source your RC file before you run the commands below.
+
+1. Create a stack by running the following command in the OpenStack CLI:
+
+    ```
+    $ openstack stack create -t basic-stack.yaml basic-stack
+    ```
+    SYNTAX:
+    ```
+    openstack stack create -t  <template name> <stack name>
+    ```
+
+    This command will return the following:
+    
+    ```
+    +---------------------+--------------------------------------+
+    | Field               | Value                                |
+    +---------------------+--------------------------------------+
+    | id                  | 6be269ec-d22f-4cf0-bf04-df71cc6dcf75 |
+    | stack_name          | basic-stack                          |
+    | description         | No description                       |
+    | creation_time       | 2019-02-12T15:58:20Z                 |
+    | updated_time        | None                                 |
+    | stack_status        | CREATE_IN_PROGRESS                   |
+    | stack_status_reason | Stack CREATE started                 |
+    +---------------------+--------------------------------------+
+    ```
+
+    
+## Next steps
+
+### Generate a heat template for your existing environment.
+Flame a standalone tool to convert from an existing environment into a heat template.
+More information is available at https://github.com/openstack/flame
+
+The following document provides more options for configuring and creating Heat templates when using UKCloud for OpenStack.
+
+[*Heat Orchestration Template (HOT) specification*](https://docs.openstack.org/heat/rocky/template_guide/hot_spec.html)
+
+## Feedback
+
+If you have any comments on this document or any other aspect of your UKCloud experience, send them to <products@ukcloud.com>.
```

Comment:
There's new copy for this feedback section:

If you find an issue with this article, click **Improve this Doc** to suggest a change. If you have an idea for how we could improve any of our services, visit [UKCloud Ideas](https://ideas.ukcloud.com). Alternatively, you can contact us at <products@ukcloud.com>.